saleor/dashboard/urls.py

```diff
@@ -33,4 +34,4 @@
     url(r'^style-guide/', core_views.styleguide, name='styleguide'),
     url(r'^search/', include(search_urls)),
     url(r'^taxes/', include(taxes_urls)),
-]
+    url(r'^next/', TemplateView.as_view(template_name='dashboard/next.html'))]
```

Comment:
We should exclude this change from this PR. For now new dashboard should be available only on `dashboard-2.0` branch.